csvcubed/csvcubed/cli/build.py

```diff
@@ -19,5 +30,50 @@ def build(
 
     # Create a path to save the validation-errors.json file in the ./out directory
 
-    _logger.debug(f"CSV: {csv_path.absolute()}")
-    _logger.debug(f"qube-config.json: {config.absolute()}")
+    _logger.debug(f"CSV: {csv_path.absolute() if csv_path is not None else ''}")
+    _logger.debug(f"qube-config.json: {config.absolute() if config is not None else ''}")
+
+    if config:
+        cube, json_schema_validation_errors = get_cube_from_config_json(config, csv_path)
+
+    else:
+        cube, json_schema_validation_errors = get_cube_from_data(csv_path)
+
+    validation_errors = cube.validate()
+    validation_errors += validate_qb_component_constraints(cube)
+
+    if not output_directory.exists():
+        print(f"Creating output directory {output_directory.absolute()}")
+        output_directory.mkdir()
+
+    if len(validation_errors) > 0 or len(json_schema_validation_errors) > 0:
+        for error in validation_errors:
+            print(
+                f"Validation Error: {error.message}"
+            )
+
+        for err in json_schema_validation_errors:
+            print(
```

Comment:
Can you output the schema validation errors as `logger.warning`?

Reply:
done